frontend/src/scenes/project/Settings/WebhookIntegration.tsx

```diff
@@ -114,6 +114,15 @@ const logic = kea<logicType<UserType>>({
     }),
 })
 
+export function AsyncActionMappingNotice(): JSX.Element {
+    return (
+        <p>
+            Please note that webhooks and actions may be delayed up to 5 minutes due to open-source PostHog
+            configuration.
+        </p>
```

Comment:
I committed this notice. I'd really like to be clear for users so that they get why there's a delay, WDYT here @mariusandra?

Reply:
Good enough for me